commands/link/index.js

```diff
@@ -19,6 +20,13 @@ class LinkCommand extends Command {
   }
 
   initialize() {
+    if (this.options.npmClient === "pnpm") {
+      throw new ValidationError(
+        "EWORKSPACES",
+        "Link is not supported with pnpm workspaces, since pnpm will automatically link dependencies during `pnpm install`. See the pnpm docs for details: https://pnpm.io/workspaces."
```

Comment:
Same comment regarding trailing period and overall consistency of error formatting for these